src/firefly/js/ui/tap/SpatialSearch.jsx

```diff
@@ -537,7 +578,7 @@ function makeUserAreaConstraint(regionOp, userArea, adqlCoordSys ) {
         return `CONTAINS(${region}, ${containedBy})=1`;
 
     } else if (regionOp === 'intersects'){
-        return `INTERSECTS(${userArea}, s_region)=1`;
+        return `INTERSECTS(s_region, ${userArea})=1`;
```

Comment:
when did you flip these parameters? Is there a standard that they were breaking? Maybe is was not working at all. Have you tested it with CADC?

Reply:
Spherex ObsTAP wasn't working with s_region as second argument. Judith told me to use it as first argument.

I checked IVOA doc: it doesn't say arguments order matter in INTERSECTS, unlike CONTAINS: https://www.ivoa.net/documents/ADQL/20210528/PR-ADQL-2.1-20210528.html#tth_sEc4.2.17

I just tested CADC obscore with both cases - the query returns with same results (number of rows, active chart, etc.)